src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -649,10 +654,10 @@ def _acquireTiles(self):
 
     def _get_z_on_focus_plane(self, x, y):
         if not self._focus_plane:
-            gamma, normal = linalg.fit_plane_lstsq(self._focus_points)
-            self._focus_plane["gamma"] = gamma
+            c, normal = linalg.fit_plane_lstsq(self._focus_points)
+            self._focus_plane["c"] = c
             self._focus_plane["normal"] = normal
-        point_on_plane = (0, 0, self._focus_plane["gamma"])  # where the plane intersects with the z-axis
+        point_on_plane = (0, 0, self._focus_plane["c"])  # where the plane intersects with the z-axis
```

Comment:
To keep it consistent with `linalg.fit_plane_lstsq`, can you name it back to `gamma`?